src/index.html

```diff
@@ -2,11 +2,324 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+   >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link logo-link">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo-link__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#notebooks"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadjets" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#notepads" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photos" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#videos" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__container">
+        <div class="product product--wrap" data-qa="card-hover">
```

Comment:
use mix class for card's margins, not modifier

```suggestion
    <main class="main">
      <div class="container main__container">
        <div class="product container__product" data-qa="card-hover">
```